src/Tools/dotnet-monitor/Auth/ApiKey/MonitorApiKeyAuthConfigurator.cs

```diff
@@ -15,15 +16,19 @@
 
 namespace Microsoft.Diagnostics.Tools.Monitor.Auth.ApiKey
 {
-    internal abstract class AbstractMonitorKeyAuthConfigurator : IAuthenticationConfigurator
+    internal sealed class MonitorApiKeyAuthConfigurator : IAuthenticationConfigurator
     {
+        private readonly GeneratedJwtKey _pinnedJwtKey;
         private readonly bool _enableNegotiation;
 
-        public AbstractMonitorKeyAuthConfigurator()
+
+        public MonitorApiKeyAuthConfigurator(GeneratedJwtKey pinnedJwtKey = null)
         {
+            _pinnedJwtKey = pinnedJwtKey;
+
             if (OperatingSystem.IsWindows())
             {
-                _enableNegotiation = true;
+                _enableNegotiation = false;
```

Comment:
I might just be missing something, but it looks like this value is always going to be false